larq_zoo/sota/quicknet.py

```diff
@@ -30,42 +31,49 @@ def LCEFirstLayer(filters: int, x: tf.Tensor) -> tf.Tensor:
     return tf.keras.layers.BatchNormalization(momentum=0.9, epsilon=1e-5)(x)
 
 
+def squeeze_and_excite(inp: tf.Tensor, filters: int, r: int = 16):
+    """Squeeze and Excite as per [Squeeze-and-Excitation Networks](https://arxiv.org/abs/1709.01507)"""
```

Comment:
We should add a reference here (or later) to https://openreview.net/forum?id=BJg4NgBKvH